contrast_security/user_api/session_metadata_api.py

```diff
@@ -9,3 +9,27 @@ def __init__(self):
     def filter_app_session_metadata(self, org_uuid, app_id):
         path = '{org_uuid}/metadata/session/{app_id}/filters'.format(org_uuid=org_uuid, app_id=app_id)
         return self._post(path, json={})
+
+    def get_app_agentsessions(self, org_uuid, app_id):
+        path = '{orgUuid}/applications/{appId}/agent-sessions'.format(org_uuid=org_uuid, app_id=app_id)
+        return self._get(path, json={})
+
+    def filter_app_session_metadata_filter(self, org_uuid, app_id, limit=None, offset=None):
+        path = '{orgUuid}/applications/{appId}/agent-sessions/filter'.format(org_uuid=org_uuid, app_id=app_id)
+
+        # Prepare query parameters
+        params = {}
+        if limit is not None:
+            params['limit'] = limit
+        if offset is not None:
+            params['offset'] = offset
+
+        return self._post(path, json={}, params=params)
```

Comment:
The json={} argument in the _post method appears to be an empty dictionary. If not needed, consider removing it to simplify the method call:
`return self._post(path, params=params)`